src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -74,6 +78,44 @@ class StacktraceLink extends AsyncComponent<Props, State> {
     }
   }
 
+  componentDidMount() {
+    this.promptsCheck();
+  }
+
+  async promptsCheck() {
+    const {organization} = this.props;
+
+    const prompt = await promptsCheck(this.api, {
+      organizationId: organization.id,
+      projectId: this.project?.id,
+      feature: 'stacktrace_link',
+    });
+
+    this.setState({
+      isDismissed: !prompt?.dismissedTime ? false : true,
+    });
+  }
+
+  dismissPrompt() {
+    const {organization} = this.props;
+    promptsUpdate(this.api, {
+      organizationId: organization.id,
+      projectId: this.project?.id,
+      feature: 'stacktrace_link',
+      status: 'dismissed',
```

Comment:
What's the difference between 'dismissed' and 'snoozed'? I assume the latter will pop back up after some period of time but the former is gone forever? If so is it gone forever for everyone or just that user?

Reply:
@ceorourke yeah exactly, gone forever vs. gone for a certain amount of time. but its per the user, so I think just using dismiss in this case is probably fine.